src/content/structured/components/alerts/code.mdx

```diff
@@ -142,18 +138,18 @@ export const snippetsWithAction = [
 export const snippetsTitleAbove = [
   {
     language: "Web component",
-    snippet: `<ic-alert heading="Title displayed above the message" message="This alert has the title at the top of the alert and the message below it." title-above="true"></ic-alert>`,
+    snippet: `<ic-alert heading="What is Coffee?" message="Coffee is a brewed drink prepared from roasted coffee beans." title-above="true"></ic-alert>`,
   },
   {
     language: "React",
-    snippet: `<IcAlert heading="Title displayed above the message" message="This alert has the title at the top of the alert and the message below it." titleAbove />`,
+    snippet: `<IcAlert heading="What is Coffee?" message="Coffee is a brewed drink prepared from roasted coffee beans." titleAbove />`,
   },
 ];
 
 <ComponentPreview snippets={snippetsTitleAbove}>
   <IcAlert
-    heading="Title displayed above the message"
-    message="This alert has the title at the top of the alert and the message below it."
+    heading="What is Coffee?"
+    message="TCoffee is a brewed drink prepared from roasted coffee beans."
```

Comment:
Can you make this message longer so that the title does appear above the message?